in-market-test/run_queries.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+IPA_REPO=$1
+NETWORK_TOML=$2
+
+query_sizes=(100 200 300)
+
+# Define the arrays
+attribution_windows=(86400 604800)
+user_cappings=(1 2 3 4)
+breakdown_keys=(32 64)
+
+DATA_PATH=~/SyntheticTest/inputs
+LOGS_PATH=~/SyntheticTest/logs
+
+
+mkdir -p $DATA_PATH
+mkdir -p $LOGS_PATH
+
+echo "Generating input data"
+cd $IPA_REPO
+for (( n=0; n<${#query_sizes[@]}; n++ )); do
+  cargo run --bin report_collector --features="cli test-fixture" -- gen-ipa-inputs -n ${query_sizes[$n]} --output-file $DATA_PATH/ipa-events-${query_sizes[$n]}.txt
```

Comment:
this is likely running in debug mode. You could also build a binary `cargo build <bin> --release` and run it instead